src/components/EarnDashboard/components/ProviderDetails/OpportunityRow.tsx

```diff
@@ -216,6 +216,7 @@ export const OpportunityRow: React.FC<
             subText={assetCellSubText}
             justifyContent='flex-start'
             isExternal={opportunity.isReadOnly}
+            opportunityName={opportunity.opportunityName}
```

Comment:
Do we still need this @NeOMakinG? i.e after discussion with @reallybeard, we want to leverage asset symbol here as well (i.e RUNE) for consistency

Reply:
Done in [600a4f2](https://github.com/shapeshift/web/pull/7540/commits/600a4f225eaef734e086b84d21d13c72255b9032)

Final result 
<img width="1074" alt="image" src="https://github.com/user-attachments/assets/c9a3df80-1b5a-473a-af52-a441c8e9362e">
w